Main.java

```diff
@@ -1,3 +1,4 @@
+import java.util.Scanner;
 /**
  *
  * @author 

```

Comment:
Don't forget to add your name and the program description into your code header!